### PR TITLE
Husan's solution

### PR DESCRIPTION
**Pull request by husan-eshonqulov**
Here I confirm that I:

- [x] ran `yarn test` before pushing changes.
- [x] wrote myself and understood all the code.
- [x] checked cleanliness of my code after finishing the implementation.

### CODE REVIEW

**Comment by husan-eshonqulov** [comment]
I have just looked at Oybek aka's code. It's almost the same as mine. 
I asked him for some help, but I didn't get his code.
Now, I am feeling like I copied 😔

File: menu.js [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+      price: dishPrice
+    };
+
+    this._courses[courseName].push(dish);
+  },
+
+  getRandomDishFromCourse(courseName) {
+    const randomIndex = Math.floor(Math.random() * this._courses[courseName].length);
+    
+    return this._courses[courseName][randomIndex];
```
Comment (bahriddin) [inline menu.js]: You are calling `this._courses[courseName]` 2 times. I think making extra variable for its value is a good idea. Something like `courseDishes` should be fine.

**Review by bahriddin — CHANGES_REQUESTED** [review]
One comment and ready to go! 👍

**Comment by bahriddin** [comment]
> I have just looked at Oybek aka's code. It's almost the same as mine.
> I asked him for some help, but I didn't get his code.
> Now, I am feeling like I copied 😔

If you can write as good as this after this task, mission is accomplished.

**Comment by husan-eshonqulov** [comment]
> > I have just looked at Oybek aka's code. It's almost the same as mine.
> > I asked him for some help, but I didn't get his code.
> > Now, I am feeling like I copied 😔
> 
> If you can write as good as this after this task, mission is accomplished.

> Ok

**Review by bahriddin — APPROVED** [review]

**Pull request closed without merging**